tekton/release-cheat-sheet.md

```diff
@@ -130,6 +130,13 @@ the pipelines repo, a terminal window and a text editor.
 14. Announce the release in Slack channels #general and #pipelines.
 
 15. Update [the catalog repo](https://github.com/tektoncd/catalog) test infrastructure
-to use the new release by updating the `RELEASE_YAML` link in [e2e-tests.sh](https://github.com/tektoncd/catalog/blob/v1beta1/test/e2e-tests.sh).
+to use the new release by updating:
+* The `RELEASE_YAML` link in [e2e-tests.sh](https://github.com/tektoncd/catalog/blob/master/test/e2e-tests.sh).
```

Comment:
Changing `RELEASE_YAML` to the new release would actually have all the tasks run against the new release, isn't it?

https://github.com/tektoncd/catalog/blob/dc4ced0942f0f9a3ad21e1dbcb8bc669e3f0d81b/test/e2e-tests.sh#L19

Reply:
yes, pointing the `RELEASE_YAML` to the new release would allow tasks/pipelines in the catalog to run against the new pipeline version.